bot.py

```diff
@@ -513,7 +513,7 @@ async def handle_team_code(self, message, shortend=False):
         if not team:
             log.debug(f'nothing found in message {message.content}')
             return
-        log.debug(f'[{message.guild.name}][{message.channel}][{message.author.display_name}] {message.content}')
+        debug(message)
```

Comment:
.guild is none. If we fix that, this log output would also be fixed.
Not sure if I missed something. But shouldnt it be "log.debug" instead of debug?

Reply:
it's a follow up to the previous commit.

Reply:
sry my fault. Gary already commited things I didnt saw.